src/tests/verification_mock/mod.rs

```diff
@@ -72,15 +72,35 @@ macro_rules! no_error {
 const ROOT1: &[u8] = include_bytes!("root1.crt");
 const ROOT1_INT1: &[u8] = include_bytes!("root1-int1.crt");
 const ROOT1_INT1_EXAMPLE_COM_GOOD: &[u8] = include_bytes!("root1-int1-ee_example.com-good.crt");
-#[cfg(any(windows, target_os = "macos", target_os = "linux"))]
+#[cfg(any(
+    windows,
+    target_os = "android",
+    target_os = "macos",
+    target_os = "linux"
+))]
```

Comment:
At this point, I believe we're now exactly matching the `cfg` guard for the entire file itself:
```

#![cfg(any(
    windows,
    target_os = "android",
    target_os = "macos",
    target_os = "linux"
))]
```

Given that future iOS and WASM testing would use an existing, fully-supported, verifier, I don't anticipate needing to re-guard these. Could we try removing all of these 4-platform `#[cfg]` guards and the ones inside the test generation macro for simplicity?

Reply:
> Could we try removing all of these 4-platform #[cfg] guards 

Good call, thanks! I was worried about the future iOS/WASM testing but as you mention that probably isn't a good reason to keep these.

> and the ones inside the test generation macro for simplicity?

I left these because the "stapled_revoked" test cases still need to be guarded to not run on Linux.

Reply:
Ah ok, that makes sense 👍